relay-dynamic-config/src/feature.rs

```diff
@@ -15,6 +15,8 @@ pub enum Feature {
     ///
     /// Enables device.class tag synthesis on mobile events.
     DeviceClassSynthesis,
+    /// Enables metric extraction from spans.
+    SpanMetricExtraction,
```

Comment:
Do we want to align this name with the feature name? 
```suggestion
    SpanMetricsExtraction,
```